resources/js/Pages/Profile.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div>
+        <Head title="Profile" />
+        <Menu />
+        <section>
+            <div class="columns">
+                <div class="column">
+                    <div class="container">
+                        <img src="http://placekitten.com/g/200/200" />
+                        <div>
+                            <h2 class="title is-2">
+                                {{
+                                    publicUser.first_name +
+                                    " " +
+                                    publicUser.last_name
+                                }}
+                            </h2>
+                            <h4 class="title is-4">{{ publicUser.email }}</h4>
+                        </div>
+                    </div>
+                    <!-- Please do not indent in another way-->
+                    <div class="box">{{ publicUser.description }}</div>
+                </div>
+
+                <div class="column">
+                    <h2 class="title is-2">Public Albums :</h2>
+                    <div
+                        v-show="publicAlbums.length > 0"
+                        class="columns is-multiline"
+                    >
+                        <div v-for="n in 4" :key="n" class="column is-half">
+                            <CardAlbum
+                                title="Vacances a Rome"
+                                username="Bedric Sila"
+                                thumbnail="https://picsum.photos/600/400"
+                            />
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <Footer />
+    </div>
+</template>
+<style scoped lang="scss">
+img {
+    max-width: 200px;
+    border-radius: 50%;
+    padding-right: 1rem;
+}
+.container {
+    display: flex;
+    align-items: center;
+}
+
+.columns {
+    padding: 2rem;
+}
+
+.box {
+    margin-top: 1rem;
+    white-space: pre-wrap;
+    box-shadow: none;
+}
```

Comment:
```suggestion
    box-shadow: none;
    border: 1px solid rgba(0,0,0,.15);
}
```
Now that the box doesn't have a shadow anymore, the contrast between the box and the background is lacking.
I suggest here a change that add a thin border around the box, like this :
![firefox_2022-01-03_19-22-22](https://user-images.githubusercontent.com/6026044/147965549-af919739-d923-4565-a619-f83c79a87ee5.png)
Before, it was like that :
![firefox_2022-01-03_19-22-32](https://user-images.githubusercontent.com/6026044/147965575-ca438c98-e835-4dd9-902e-6ff6a42ff685.png)

Feel free to reject/change this suggestion if you have a better idea.